cluster/images/hyperkube/turnup.sh

```diff
@@ -20,7 +20,7 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-K8S_VERSION=${K8S_VERSION:-"1.2.0-alpha.7"}
+K8S_VERSION=${K8S_VERSION:-"1.2.0"}
```

Comment:
This is doomed to be out of dat forever. We should avoid embedding k8s_version into scripts and instead discover the most recent